### ISSUE TITLE
[BUG] Connection Issues with v3

### ISSUE THREAD

**Issue by kajatv** [bitfocus/companion-module-softron-ontheairvideo#8]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hello,  I have several installs of Bitfocus where I use it to control a Softron On the Air Video playlist.  On the newest release of the software when I try to make a connection to my Softron the connection just goes into a continuous connect disconnect loop.  I had to down rev to version 2.4.2 before I could get a stable connection.  Any Ideas?

### Steps To Reproduce

None. Cannot achieve a stable connection on newest Companion version.

### Environment (please complete the following information)

```markdown
- OS:Mac Ventura 13
- Browser:google Chrome
- Companion Version:3.1.1
```

**Comment by bryce-seifert** [comment]
Which version number of OnTheAir video are you using? 
Can you verify in your module config that the IP and port are correct?
What is the specific error logged in the Companion log tab?

**Comment by kalelzone** [comment]
They are using 4.2 of OTAV

**Comment by bryce-seifert** [comment]
Any other of the requested info you can provide?

**Comment by kalelzone** [comment]
After testing for a day, it keeps its connection if you open OTAV after Bitfocus is launched. If you launch Bitfocus after OTAV is launched it loses connection. Same with Movie Recorder.

**Comment by sphlabs** [comment]
Do you have any comments in your playlist? There was a bug where this would cause crashes in the module.

**Comment by kalelzone** [comment]
yes, but this is even happening when there are no comment lines.

**Comment by kalelzone** [comment]
it happens so fast when the bug is triggered you almost miss it. I added a button to display error state on devices and it flashes from 0 to 1 very fast.

**Comment by kalelzone** [comment]
OK, i have more info. If a Playlist is open with comment lines then the connection flashes when you connect, if you close the playlist with comment lines in it and reopen it it stops disconnecting.

**Comment by kalelzone** [comment]
Curious if the Comment bug is a softron issue or bitfocus module bug

**Comment by sphlabs** [comment]
It's a bug in the Companion module (wasn't properly handling cues with no duration).

I've just pushed an update that will hopefully fix this for you. Should be in the beta release in a day or so.

**Comment by kalelzone** [comment]
Thank you

**Comment by kalelzone** [comment]
This is fixed but now its not pulling the Clip Variables....
![Screenshot 2024-01-26 at 9 34 14 AM](https://github.com/bitfocus/companion-module-softron-ontheairvideo/assets/132862470/f424b250-bdf7-4f50-8cb3-b37febf45082)

**Comment by sphlabs** [comment]
Can you confirm what version of the module you are using now?

**Comment by kalelzone** [comment]
6693, this mornings build

**Comment by sphlabs** [comment]
I just checked, that is still running the older OTA module version. Still waiting on the core team to integrate my latest bug-fix release.

**Comment by bryce-seifert** [comment]
Fixed with https://github.com/bitfocus/companion-module-softron-ontheairvideo/commit/e0bbc4de9f03a3f7d8c093888ca17701e683ef93
